### PR TITLE
Generalize db setup, reproject bug fix

### PR DESCRIPTION
**Pull request by loicgasser**
FYI: populating new data on big DB cluster

### CODE REVIEW

**Comment by gjn** [comment]
All data uploaded?

File: database.cfg.mako [hunk 1]
```diff
@@ -7,28 +7,28 @@ user: pgkogis
 password: ${pgpass}
 
 [Database]
-name: forge
+name: forge_main
```
Comment (gjn) [inline database.cfg.mako]: So we use a new db?

**Comment by gjn** [comment]
I think we need an easy way to update an existing table with new data that only partially covers new extent. 

Example: let's assume we have a table filled for whole switzerland. Then we recieve a new delivery for a give extent that is only a subset. The we would need to replace the existing shapes for this extent with the new ones.

**Comment by gjn** [comment]
Or it might be more reasonable to say: each delivery is a new db.

File: database.cfg.mako (hunk 1 above)
Comment (loicgasser) [inline database.cfg.mako]: yes we keep the old one for now. So that we can compare/test different confs.

**Comment by loicgasser** [comment]
data upload is read up to 8m, at 4m we are at around 80%. 40% for 2m and around 20% and 10% for 1 and 0.5m.

You can already update an existing DB with this PR.
See:
https://github.com/geoadmin/3d-forge/pull/127/files#diff-b7095541335f63fb8d07745c2035e7dcR272
and
https://github.com/geoadmin/3d-forge/pull/127/files#diff-b7095541335f63fb8d07745c2035e7dcR416


So basically edit the generated `database.cfg` with the folder with the new data and the tablename.
Then `make populate`

The table gets created if it doesn't exist or if it exists it simply adds new features.

**Comment by gjn** [comment]
very nice.

pep probably not happy.

**Comment by loicgasser** [comment]
pep probably not happy.
Yes must remove an import

Data already in DB for 128m to 8m. It's pretty fast. (with reprojection)

@gjn Can you open a few pipes on winscp for 1m and 0.5m so that we have all the data on Monday?

Make sure you don't replace existing files (No to all)

**Comment by gjn** [comment]
YEs, when I'm in office tomorrow. Send me the source path.

**Comment by gjn** [comment]
@loicgasser Can you send me the source path...can't find any shapefiles.

**Comment by loicgasser** [comment]
\\prod\kogis\iwi\3DTerrain\

**Comment by loicgasser** [comment]
populating zoom 4 right now.

(SO en prio  et ensuite SE dans un autre fichier qu'on mergera à la fin)

**Comment by loicgasser** [comment]
sorry SW but make sure we have all the files (5'000 +)

**Comment by gjn** [comment]
So current is SW or SO? (I've found SO files in 0.5)...

**Comment by loicgasser** [comment]
SO

**Comment by loicgasser** [comment]
2015-08-29 03:08:40,165 | forge.lib.tiler                  | INFO     | It took 11:51:08.614790 to create 1109144 tiles (706297 were skipped)

**Pull request merged**